wdl/ReshardVcf.wdl

```diff
@@ -3,20 +3,19 @@ version 1.0
 import "Structs.wdl"
 import "TasksMakeCohortVcf.wdl" as MiniTasks
 
-# Consumes a contig-sharded vcf where some shards may contain a small proportion of records from other
-#  contigs and moves them to their correct shards.
+# Consumes a set of vcfs and distributes them into contig shards
 
 workflow ReshardVcf {
   input {
-    Array[File] vcfs  # Must be sorted and indexed
+    Array[File] vcfs  # Order does not matter but must be sorted and indexed
     File contig_list
     String prefix
     Boolean? use_ssd
     String sv_base_mini_docker
     RuntimeAttr? runtime_override_reshard
   }
 
-  Array[String] contigs = read_lines(contig_list)
+  Array[String] contigs = transpose(read_tsv(contig_list))[0]
```

Comment:
Make sure to update the ReshardVcf JSON template to use the primary_contigs_fai to match this change - it looks like it was initially set up with primary_contigs_list

Reply:
It runs ok with the contigs list too